spec/defines/dns_logging_channel_spec.rb

```diff
@@ -163,7 +174,7 @@
           }
         end
 
-        it { is_expected.not_to compile }
+        it { is_expected.to compile.with_all_deps }
```

Comment:
I generally like to have more specific validation, something like
```suggestion
        it { is_expected.to compile.with_all_deps }
        it { is_expected.to contain_concat__fragment("named.conf-logging-channel-#{title}.dns").with_content(%r{file "/path/to/file.log" size 5m;});
```